pkgs/dart_services/test/server_v3_test.dart

```diff
@@ -79,6 +80,8 @@ class MyApp extends StatelessWidget {
 
       expect(result, isNotNull);
       expect(result.issues, isEmpty);
+      expect(result.packageImports, isNotEmpty);
```

Comment:
Why check this when checking for `flutter` in the next line?

Reply:
Yeah, its redundant; the intent was to test that we parsed some imports, and then that we parsed the correct imports, but we could just as well only do the 2nd test.